app/assets/stylesheets/new_board/_side_bar.scss

```diff
@@ -0,0 +1,64 @@
+.SideBar {
+  display: inline-block;
+  width: 40px;
+
+  &__link {
+    width: 100%;
+    height: 40px;
+    position: relative;
+    border-radius: 5px;
+    -webkit-transition: background-color ease-in-out .2s;
+    -o-transition: background-color ease-in-out .2s;
```

Comment:
We probably don't need that because we use autoprefixer https://github.com/Codeminer42/cm42-central/blob/master/Gemfile#L15.

Remove it from the code (leaving only the one without the prefix), run it, and check if the prefixed version will be there (you can do it inspecting the element).